proof/invariant-abstract/CNodeInv_AI.thy

```diff
@@ -902,70 +897,99 @@ lemma not_recursive_cspaces_irq_state_independent[intro!, simp]:
 
 context CNodeInv_AI begin
 
+(* FIXME: Move. Is this safe in wp_comb? *)
```

Comment:
`wp_comb` has potentially large performance impact, but it looks like straight `wp` might work here. Have you tried that?

Reply:
I did try that but it didn't work. 

Since it's apparently wasn't needed until now, I can explicitly add it into the line where I use it and avoid any potential performance impact.

Reply:
That would work. If you want to, we could also measure if there is any real impact by running AInvs+Refine with and without -- if there is no clear difference there, then it should be fine. The `wb_comb` implementation changed a few times, so I might be over-cautious here.

Reply:
I'm sure it was useful at one point, but now it seems to be unused so assuming tests all pass we can forget about it.